test/PowerShellEditorServices.Test/Language/TokenOperationsTests.cs

```diff
@@ -111,23 +114,30 @@ public class TokenOperationsTests
 # Comment Block 2
 # Comment Block 2
 $something = $true
-#endregion Comment Block 3";
+#endregion Comment Block 3
+
+# What about anonynous variable assignment
+${this
```

Comment:
This is the edge case I like to see. Allowing these awful variable names is one of @JamesWTruher's proudest additions to PowerShell :)